src/test-apps/ToolCommonOptions.cpp

```diff
@@ -823,7 +823,7 @@ WEAVE_ERROR ServiceDirClientOptions::GetRootDirectoryEntry(uint8_t *buf, uint16_
 
     Write8(buf, 0x41);                                       // Entry Type = Host/Port List, Host/Port List = 1
     LittleEndian::Write64(buf, kServiceEndpoint_Directory);  // Service Endpoint Id = Directory Service
-    Write8(buf, 0x80);                                       // Host ID Type = Fully Qualified, Suffix Index Present = false, Port Id Present = true
+    Write8(buf, 0x08);                                       // Host ID Type = Fully Qualified, Suffix Index Present = false, Port Id Present = true
```

Comment:
We messed this up once, lets fix it for real this time: 
line 824:

    uint8_t hostPortListLength = 1;
    Write8(buf,  kDirectoryEntryType_HostPortList | (hostPortListLength & kMask_HostPortListLen));

line 826: 

     Write8(buf, kHostIdType_FullyQualified |  kMask_PortIdPresent /* | kMask_SuffixIndexPresent);

Reply:
Done, thanks